src/components/scenes/EditTokenScene.tsx

```diff
@@ -197,7 +197,7 @@ function EditTokenSceneComponent(props: Props) {
  * In the future this scene may need to handle other weird networks
  * where the networkLocation has other contents.
  */
-const asMaybeContractLocation = asMaybe(asObject({ contractAddress: asString }))
+export const asMaybeContractLocation = asMaybe(asObject({ contractAddress: asString }))
```

Comment:
Might be a good time to move /aave/cleaners/asEthTokenContractAddress.ts to a
common location and add asMaybeContractLocation to it

Reply:
Will do for the full Thorchain Savers with token task.